backend/app/tests/integration/api/endpoints/test_version.py

```diff
@@ -1,21 +1,26 @@
 import pytest
 import requests
 
+from tests.utils import validate_datetime
+
+
 ENDPOINT = "version"
 
 
 @pytest.mark.integration
-@pytest.mark.nondestructive
-def test_version_get_request(api_url):
+def test_version_get_request(api_url, tag, stack_name, revision):
```

Comment:
~~ Do we need a `@pytest.mark.parametrize( "api_url, tag, stack_name, revision", ("dev","dev","dev","20210101.1111111")],)`here? ~~
Actually  Nevermind, I've seen the `conftest.py`.